frontend/src/reducers/user.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const user = createSlice({
+    name: 'user',
+    initialState: {
+        username: null,
+        accessToken: null,
+        errors: null
+    },
+    reducers: {
+        setUsername: (store, action) => {
+            store.username = action.payload;
+        },
+        setAccessToken: (store, action) => {
+            store.accessToken = action.payload;
+        },
+        setErrors: (store, action) => {
+            store.errors = action.payload;
+        }
+    }
+});
+
+export default user;
```

Comment:
You need to store your data in the local storage to avoid logging out your user after refreshing the page.

Reply:
This should be added now 👍